src/cascadia/TerminalControl/SearchBoxControl.cpp

```diff
@@ -0,0 +1,152 @@
+﻿//
+// SearchBoxControl.cpp
+// Implementation of the SearchBoxControl class
+//
+
+#include "pch.h"
+#include "SearchBoxControl.h"
+#include "SearchBoxControl.g.cpp"
+
+using namespace winrt;
+using namespace winrt::Windows::UI::Xaml;
+
+namespace winrt::Microsoft::Terminal::TerminalControl::implementation
+{
+    SearchBoxControl::SearchBoxControl() :
+        _goForward(false),
+        _isCaseSensitive(false)
+    {
+        InitializeComponent();
+
+        _goForwardButton = this->FindName(L"SetGoForwardButton").try_as<Controls::Primitives::ToggleButton>();
+        _goBackwardButton = this->FindName(L"SetGoBackwardButton").try_as<Controls::Primitives::ToggleButton>();
+
+        // TO DO: Is there a general way to put all the focusable elements in the

```

Comment:
Bare TODO. Needs a workitem generated and listed or to be removed before commit.

Reply:
The TODO is still here.

Reply:
Sorry I did not reply this. I thought about his and I would just remove this TO DO. The search box is not a big dialog box so I think it makes sense to manually add the navigable elements into the hash set.